packages/react/.gemini.js

```diff
@@ -21,7 +21,8 @@ module.exports = {
     chrome: {
       desiredCapabilities: {
         browserName: "chrome",
-        version: "54.0",
+        version: "65.0",
+        windowSize: "1024x768",
```

Comment:
gemini was failing because the window size was not being specified. This seems to happen above Chrome version 60.